app/importers/data_transformers/csv_transformer.rb

```diff
@@ -9,9 +9,11 @@ def initialize(options={})
   end
 
   def transform(file)
-    csv = CSV.parse(file, headers: @headers,
-                          header_converters: :symbol,
-                          converters: lambda { |h| nil_converter(h) })
+    cleaned_file = file.gsub("\\\"", "")
```

Comment:
The tests are good but seems worth a comment here.

Thinking aloud (feel free to ignore), would the `liberal_parsing` option described in https://github.com/ruby/ruby/pull/1160 work here too?  I see the description is somewhat different, but seems like it might be worth trying (test will confirm if it works or changes any other parsing)).  Alternately https://github.com/intercom/hippie_csv might be worth trying to drop in to see if it works w/o other changes.

Reply:
@kevinrobinson Cool, thank you, will take a look at those! 👀 😎

Reply:
@kevinrobinson So HippieCSV looked really interesting and promising. Their description of the use case they focus on sounds really similar to ours. I made a new branch to try it out. One issue I ran into upfront was that I couldn't find any support for headers: https://github.com/intercom/hippie_csv/search?utf8=%E2%9C%93&q=headers&type=

That makes me think HippieCSV could be a good future direction, but isn't a drop-in replacement. To make it work we'd have to stop reading headers from the exported CSVs and pass them in ourselves as hardcoded arrays for each of the files we parse.

Reply:
Looking at `liberal_parsing` next: https://blog.bigbinary.com/2016/11/22/ruby-2-4-introduces-liberal_parsing-option-for-parsing-bad-csv-data.html, going to download ruby 2.4, do a little reading on what the changes are in 2.4, and see if this works as a drop in replacement.

Reply:
Alrighty, so #1402 uses Ruby 2.4 and tries `liberal_parsing`. The specs fail, kind of interesting why: 

The blog post above from BigBinary explains that liberal_parsing makes the CSV tolerate errors like "CSV::MalformedCSVError: Illegal quoting in line 1."

In the Travis build (https://travis-ci.org/studentinsights/studentinsights/builds/332371960), we get "CSV::MalformedCSVError: Missing or stray quote in line 3"

Maybe I'm reading too much into the differences between these error messages, but it kind of seems like the message that `liberal_parsing` fixes is about quotes not conforming to the CSV spec ("illegal"), whereas the message we're getting is coming from a deeper level and is saying the CSV library doesn't know how to parse.

Reply:
Here's the source in `ruby/csv.rb` where the two different error messages come from: https://github.com/ruby/ruby/blob/0593c6a510918ce0b6610d3f0d25748041658a55/lib/csv.rb#L1888